components/server/src/prebuilds/github-app.ts

```diff
@@ -122,6 +122,7 @@ export class GithubApp {
                         requestKind: "probot",
                         requestMethod: req.path,
                         signal: new AbortController().signal,
+                        headers: toHeaders(req.headers),
```

Comment:
is it necesary here? cc @geropl

Reply:
i.e do we need to use region provided by callback?

Reply:
or idea that we should always capture them? i just wonder whether we can affect the contract somehow if we do it

Reply:
I don't think it's necessary here, but also should not break anything. We don't optimize prebuild's region selection IIRC, and as we always received GitHub events in us-west1, that's where we historically execute them anyway. :+1: